### PR TITLE
added signup, login html and css in auth folder

### PR DESCRIPTION
**Pull request by ashishkatiyar2404**
Screenshot (798)
Screenshot (799)

### CODE REVIEW

File: authentication/login.css [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+}
+
+.text__field input:focus ~ span::before,
+.text__field input:valid ~ span::before {
+  width: 100%;
+}
+
+.pass {
+  margin: -0.5rem 0rem 2rem 0.5rem;
+  color: var(--forgot-already-color);
```
Comment (ShrutiLakhani) [inline authentication/login.css:102]: Here also. So that its easily reusable later on

**Review by ShrutiLakhani — APPROVED** [review]
Clean code. Good job! Good to merge

File: authentication/login.css [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+  border: none;
+  background: none;
+  outline: none;
+}
+
+.text__field label {
+  position: absolute;
+  top: 50%;
+  left: 0.5rem;
+  color: var(--adadad-color);
```
[the remark this answers is not in the document]
Comment (ashishkatiyar2404) [reply authentication/login.css:71]: okay

File: authentication/login.css (hunk 1 above)
Comment (ashishkatiyar2404) [reply authentication/login.css:102]: okay

File: authentication/login.css [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+  text-align: center;
+}
+
+.center form {
+  padding: 0rem 4rem;
+}
+
+form .text__field {
+  position: relative;
+  border-bottom: 2px solid var(--adadad-color);
```
Comment (pradipgharge) [inline authentication/login.css:53]: can use a better color name here !

File: authentication/login.html [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+            </li>
+            <li>
+              <i class="fa-solid fa-cart-arrow-down num">
+                <span>3</span>
+              </i>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header> -->
```
Comment (pradipgharge) [inline authentication/login.html:43]: Try not to push commented code!

File: authentication/login.css [hunk 5]
```diff
@@ -0,0 +1,143 @@
+@import url("https://fonts.googleapis.com/css2?family=Noto+Sans:wght@700&family=Poppins:wght@400;500;600&display=swap");
+
+:root {
+  --white-color: rgb(255, 255, 255);
+  --white-color-shade: rgb(233, 244, 251);
+  --silver-color: rgb(192, 192, 192);
```
Comment (dev-ashishsethi) [inline authentication/login.css:6]: hex code color could be used

File: authentication/login.css [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+  position: relative;
+}
+
+.center {
+  width: 40rem;
+  border-radius: 1rem;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
```
Comment (dev-ashishsethi) [inline authentication/login.css:35]: could have used flex box and it would have centered in 3 lines of code

File: authentication/login.css [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+  transition: 0.5s;
+}
+
+.text__field span::before {
+  content: "";
+  position: absolute;
+  top: 4rem;
+  left: 0;
+  width: 0%;
+  height: 0.2rem;
```
Comment (dev-ashishsethi) [inline authentication/login.css:84]: for less than 1 rem lengths use pixels

File: authentication/login.css [hunk 8]
```diff
@@ -0,0 +1,143 @@
...
+  background: var(--btn-auth-color);
+  transition: 0.5s;
```
Comment (dev-ashishsethi) [inline authentication/login.css:86]: 0.2 second of transition is natural for human eyes

**Review by dev-ashishsethi — APPROVED** [review]
looks good. please have a look at comments

File: authentication/signup.css [hunk 9]
```diff
@@ -0,0 +1,154 @@
...
+  text-align: center;
+}
+
+.center form {
+  padding: 0rem 4rem;
+}
+
+form .text__field {
+  position: relative;
+  border-bottom: 2px solid var(--adadad-color);
```
[the remark this answers is not in the document]
Comment (ashishkatiyar2404) [reply authentication/signup.css:53]: sure. trying to improve my naming convention😅

File: authentication/login.css (hunk 3 above)
Comment (ashishkatiyar2404) [reply authentication/login.css:53]: ok

File: authentication/login.css (hunk 5 above)
Comment (ashishkatiyar2404) [reply authentication/login.css:6]: some says if you use rgb use all rgb and vice versa for hex(and other code color also).
thnks will keep this in mind🙂

File: authentication/login.css (hunk 6 above)
Comment (ashishkatiyar2404) [reply authentication/login.css:35]: Ohh ok. thanks that doesn't come to my mind when I was coding this page. will try to chnge💯

File: authentication/login.css (hunk 7 above)
Comment (ashishkatiyar2404) [reply authentication/login.css:84]: ok

File: authentication/login.css (hunk 8 above)
Comment (ashishkatiyar2404) [reply authentication/login.css:86]: its 0.5s😶

**Pull request merged**